pkg/codefresh/argo_runtime.go

```diff
@@ -52,6 +52,40 @@ func newArgoRuntimeAPI(codefresh *codefresh) IRuntimeAPI {
 	return &argoRuntime{codefresh: codefresh}
 }
 
+func (r *argoRuntime) Create(ctx context.Context, opts *model.InstallationArgs) (*model.RuntimeCreationResponse, error) {
```

Comment:
you can see how `InstallationArgs` is a problematique name, please change to `RuntimeInstallationArgs`